src/app/components/common/AppBar/components/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -7,62 +7,55 @@ import WalletStore from '~app/common/stores/Abstracts/Wallet';
 import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application.store';
 import { useStyles } from './ConnectWalletButton.styles';
 
-type ConnectWalletButtonProps = {
-    buttonComponent?: any;
-    buttonText?: string;
-};
-
-const ConnectWalletButton = (props: ConnectWalletButtonProps) => {
-    const { buttonComponent, buttonText } = props;
-    buttonComponent;
-    buttonText;
-    const stores = useStores();
-    const walletStore: WalletStore = stores.Wallet;
-    const applicationStore: ApplicationStore = stores.Application;
-    const classes = useStyles({
-        walletConnected: walletStore.connected,
-        whiteAppBar: applicationStore.whiteNavBarBackground,
-    });
+const ConnectWalletButton = () => {
+  const stores = useStores();
+  const walletStore: WalletStore = stores.Wallet;
+  const applicationStore: ApplicationStore = stores.Application;
+  const classes = useStyles({
+    walletConnected: walletStore.connected,
+    whiteAppBar: applicationStore.whiteNavBarBackground,
+  });
 
-    const onClick = () => {
-      if (walletStore.connected) {
-        return applicationStore.showWalletPopUp(true);
-      }
-      return walletStore.connect();
-    };
+  const onClick = () => {
+    if (walletStore.connected) {
+      return applicationStore.showWalletPopUp(true);
+    }
+    return walletStore.connect();
+  };
 
-    let icon;
-    if (walletStore.wallet?.label) {
-        switch (walletStore.wallet.label) {
-            case 'Ledger':
-                icon = getImage('wallets/metamask.svg');
-                break;
-            case 'Trezor':
-                icon = getImage('wallets/metamask.svg');
-                break;
-            default:
-                icon = getImage('wallets/metamask.svg');
-                break;
-        }
+  let icon;
+  if (walletStore.wallet?.label) {
+    switch (walletStore.wallet.label) {
+      case 'Ledger':
+        icon = getImage('wallets/metamask.svg');
+        break;
+      case 'Trezor':
+        icon = getImage('wallets/metamask.svg');
+        break;
+      default:
+        icon = getImage('wallets/metamask.svg');
```

Comment:
This switch case can be simplified to have only one default section with metamask icon. Because anyway it uses metamask icon.
Also, maybe switch case can be also removed and only metamask icon will be used if suppose to be the only